service/routes.go

```diff
@@ -22,7 +22,7 @@ func (svc *Service) publicEndpoints(ctx context.Context) {
 	log.Info(ctx, "enabling public endpoints")
 
 	// Routes
-	populationTypes := handler.NewPopulationTypes(svc.responder, svc.cantabularClient)
+	populationTypes := handler.NewPopulationTypes(svc.Config, svc.responder, svc.cantabularClient, svc.datasetAPIClient)
```

Comment:
would rather break each var than only half the vars

Reply:
Not sure what you mean?